Python/qled/qled.py

```diff
@@ -0,0 +1,129 @@
+from PySide6.QtWidgets import QAbstractButton
+from PySide6.QtCore import QTimer, Qt, QByteArray
+from PySide6.QtGui import QColor, QPainter
+from PySide6.QtSvg import QSvgRenderer
+import importlib.resources
+from pathlib import Path
+from enum import Enum
+
+
+class QLed(QAbstractButton):
+
+    class ledShape(Enum):
+        Circle = "circle.svg"
```

Comment:
In the C++ variant this enum contains integer values while we use strings here. Does this still maintain compatibility to the UI files when they are compiled to Python code?